config/toolchain.in

```diff
@@ -333,4 +333,15 @@ config TOOLCHAIN_ENABLE_NLS
     help
       Say 'Y' here to enable native language support (nls).
 
+config TOOLCHAIN_CMAKE_TOOLCHAIN_FILE
+    bool
+    prompt "Install cmake toolchain file"
+    default y
```

Comment:
I wonder if we should have this default n (or actually nothing as n is the default default). Or `default y if EXPERIMENTAL`.  That's not a strong opinion, I don't often use cmake this way. What do others think?